2024-09-06 23:38:56,748 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:690]
2024-09-06 23:38:56,759 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:878]
2024-09-06 23:38:56,760 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:859]
2024-09-06 23:38:56,772 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:828]
2024-09-06 23:38:56,777 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:233]
2024-09-06 23:38:56,778 - ERROR - Unexpected error in check_expirty:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:239]
2024-09-06 23:38:56,785 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:528]
2024-09-06 23:38:56,792 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:631]
2024-09-06 23:38:56,797 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:269]
2024-09-06 23:38:56,815 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:455]
2024-09-06 23:38:56,820 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:494]
2024-09-06 23:38:56,822 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:482]
2024-09-06 23:38:56,825 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:494]
2024-09-06 23:38:56,839 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:261]
2024-09-06 23:38:56,848 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:195]
2024-09-06 23:38:56,850 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:207]
2024-09-06 23:38:56,854 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:269]
2024-09-06 23:38:56,855 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:333]
2024-09-06 23:38:56,860 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:365]
2024-09-06 23:38:56,862 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:269]
2024-09-06 23:38:56,863 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:377]
2024-09-06 23:38:56,869 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:552]
2024-09-06 23:38:56,871 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:560]
2024-09-06 23:38:56,875 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:598]
2024-09-06 23:38:56,884 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='2497800230128'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:162]
2024-09-06 23:38:56,886 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:111]
2024-09-06 23:38:56,887 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:111]
2024-09-06 23:38:56,891 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:111]
2024-09-06 23:38:56,896 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:123]
2024-09-06 23:38:56,897 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:170]
2024-10-07 16:56:49,439 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:495]
2024-10-07 16:56:49,929 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:495]
2024-10-07 17:05:39,179 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:495]
2024-10-07 17:32:41,852 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:705]
2024-10-07 17:32:41,863 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:893]
2024-10-07 17:32:41,866 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:874]
2024-10-07 17:32:41,880 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:843]
2024-10-07 17:32:41,889 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:248]
2024-10-07 17:32:41,890 - ERROR - Unexpected error in check_expirty:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:254]
2024-10-07 17:32:41,899 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:543]
2024-10-07 17:32:41,908 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:646]
2024-10-07 17:32:41,915 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:284]
2024-10-07 17:32:41,942 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:470]
2024-10-07 17:32:41,948 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:509]
2024-10-07 17:32:41,949 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:497]
2024-10-07 17:32:41,954 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:509]
2024-10-07 17:32:41,968 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:276]
2024-10-07 17:32:41,979 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:210]
2024-10-07 17:32:41,980 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:222]
2024-10-07 17:32:41,984 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:284]
2024-10-07 17:32:41,985 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:348]
2024-10-07 17:32:41,992 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:380]
2024-10-07 17:32:41,993 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:284]
2024-10-07 17:32:41,995 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:392]
2024-10-07 17:32:42,001 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:567]
2024-10-07 17:32:42,003 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:575]
2024-10-07 17:32:42,007 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:613]
2024-10-07 17:32:42,015 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='1514232286000'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:175]
2024-10-07 17:32:42,018 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:124]
2024-10-07 17:32:42,020 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:124]
2024-10-07 17:32:42,023 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:124]
2024-10-07 17:32:42,029 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:136]
2024-10-07 17:32:42,030 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:183]
2024-10-11 15:57:09,563 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:709]
2024-10-11 15:57:09,573 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:897]
2024-10-11 15:57:09,575 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:878]
2024-10-11 15:57:09,588 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:847]
2024-10-11 15:57:09,595 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:252]
2024-10-11 15:57:09,596 - ERROR - Unexpected error in check_expirty:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:258]
2024-10-11 15:57:09,603 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:547]
2024-10-11 15:57:09,610 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:650]
2024-10-11 15:57:09,617 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:288]
2024-10-11 15:57:09,643 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:474]
2024-10-11 15:57:09,647 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:513]
2024-10-11 15:57:09,650 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:501]
2024-10-11 15:57:09,654 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:513]
2024-10-11 15:57:09,669 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:280]
2024-10-11 15:57:09,681 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:210]
2024-10-11 15:57:09,682 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:226]
2024-10-11 15:57:09,687 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:288]
2024-10-11 15:57:09,688 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:352]
2024-10-11 15:57:09,694 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:384]
2024-10-11 15:57:09,696 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:288]
2024-10-11 15:57:09,697 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:396]
2024-10-11 15:57:09,704 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:571]
2024-10-11 15:57:09,707 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:579]
2024-10-11 15:57:09,712 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:617]
2024-10-11 15:57:09,728 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='2958615196976'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:175]
2024-10-11 15:57:09,730 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:124]
2024-10-11 15:57:09,734 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:124]
2024-10-11 15:57:09,738 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:124]
2024-10-11 15:57:09,744 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:136]
2024-10-11 15:57:09,746 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:183]
2024-10-22 15:03:30,788 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:714]
2024-10-22 15:03:30,798 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:907]
2024-10-22 15:03:30,801 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:888]
2024-10-22 15:03:30,815 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:857]
2024-10-22 15:03:30,822 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:253]
2024-10-22 15:03:30,823 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:259]
2024-10-22 15:03:30,832 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:552]
2024-10-22 15:03:30,838 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:655]
2024-10-22 15:03:31,012 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-22 15:03:31,048 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:477]
2024-10-22 15:03:31,053 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:516]
2024-10-22 15:03:31,055 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:504]
2024-10-22 15:03:31,058 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:516]
2024-10-22 15:03:31,073 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:281]
2024-10-22 15:03:31,083 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:211]
2024-10-22 15:03:31,084 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:227]
2024-10-22 15:03:31,088 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-22 15:03:31,089 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:355]
2024-10-22 15:03:31,096 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:387]
2024-10-22 15:03:31,112 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-22 15:03:31,113 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:399]
2024-10-22 15:03:31,120 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:576]
2024-10-22 15:03:31,123 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:584]
2024-10-22 15:03:31,126 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:622]
2024-10-22 15:03:31,135 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='2122775475168'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:176]
2024-10-22 15:03:31,137 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-22 15:03:31,139 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-22 15:03:31,143 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-22 15:03:31,148 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:137]
2024-10-22 15:03:31,150 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:184]
2024-10-22 15:04:52,136 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:714]
2024-10-22 15:04:52,145 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:907]
2024-10-22 15:04:52,147 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:888]
2024-10-22 15:04:52,161 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:857]
2024-10-22 15:04:52,167 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:253]
2024-10-22 15:04:52,168 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:259]
2024-10-22 15:04:52,177 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:552]
2024-10-22 15:04:52,183 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:655]
2024-10-22 15:04:52,327 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-22 15:04:52,361 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:477]
2024-10-22 15:04:52,365 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:516]
2024-10-22 15:04:52,367 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:504]
2024-10-22 15:04:52,370 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:516]
2024-10-22 15:04:52,385 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:281]
2024-10-22 15:04:52,411 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:211]
2024-10-22 15:04:52,413 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:227]
2024-10-22 15:04:52,417 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-22 15:04:52,418 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:355]
2024-10-22 15:04:52,423 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:387]
2024-10-22 15:04:52,426 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-22 15:04:52,427 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:399]
2024-10-22 15:04:52,433 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:576]
2024-10-22 15:04:52,437 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:584]
2024-10-22 15:04:52,441 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:622]
2024-10-22 15:04:52,450 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='2381138135664'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:176]
2024-10-22 15:04:52,452 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-22 15:04:52,454 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-22 15:04:52,458 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-22 15:04:52,463 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:137]
2024-10-22 15:04:52,465 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:184]
2024-10-22 15:06:06,190 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:714]
2024-10-22 15:06:06,201 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:907]
2024-10-22 15:06:06,203 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:888]
2024-10-22 15:06:06,215 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:857]
2024-10-22 15:06:06,221 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:253]
2024-10-22 15:06:06,222 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:259]
2024-10-22 15:06:06,232 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:552]
2024-10-22 15:06:06,239 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:655]
2024-10-22 15:06:06,245 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-22 15:06:06,268 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:477]
2024-10-22 15:06:06,272 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:516]
2024-10-22 15:06:06,273 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:504]
2024-10-22 15:06:06,278 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:516]
2024-10-22 15:06:06,292 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:281]
2024-10-22 15:06:06,302 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:211]
2024-10-22 15:06:06,303 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:227]
2024-10-22 15:06:06,308 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-22 15:06:06,309 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:355]
2024-10-22 15:06:06,316 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:387]
2024-10-22 15:06:06,317 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-22 15:06:06,319 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:399]
2024-10-22 15:06:06,325 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:576]
2024-10-22 15:06:06,328 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:584]
2024-10-22 15:06:06,332 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:622]
2024-10-22 15:06:06,341 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='1806236259760'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:176]
2024-10-22 15:06:06,343 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-22 15:06:06,360 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-22 15:06:06,365 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-22 15:06:06,371 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:137]
2024-10-22 15:06:06,372 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:184]
2024-10-31 17:05:14,363 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:714]
2024-10-31 17:05:14,373 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:907]
2024-10-31 17:05:14,376 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:888]
2024-10-31 17:05:14,388 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:857]
2024-10-31 17:05:14,395 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:253]
2024-10-31 17:05:14,397 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:259]
2024-10-31 17:05:14,404 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:552]
2024-10-31 17:05:14,411 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:655]
2024-10-31 17:05:14,418 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-31 17:05:14,440 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:477]
2024-10-31 17:05:14,444 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:516]
2024-10-31 17:05:14,446 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:504]
2024-10-31 17:05:14,449 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:516]
2024-10-31 17:05:14,463 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:281]
2024-10-31 17:05:14,474 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:211]
2024-10-31 17:05:14,475 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:227]
2024-10-31 17:05:14,480 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-31 17:05:14,481 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:355]
2024-10-31 17:05:14,487 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:387]
2024-10-31 17:05:14,489 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:289]
2024-10-31 17:05:14,490 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:399]
2024-10-31 17:05:14,497 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:576]
2024-10-31 17:05:14,500 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:584]
2024-10-31 17:05:14,504 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:622]
2024-10-31 17:05:14,513 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='1316596589072'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:176]
2024-10-31 17:05:14,514 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-31 17:05:14,516 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-31 17:05:14,520 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:125]
2024-10-31 17:05:14,525 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:137]
2024-10-31 17:05:14,527 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:184]
2024-11-04 14:48:11,221 - ERROR - Unexpected error in userData function: name 'json_data' is not defined [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:557]
2024-11-04 14:48:26,008 - ERROR - Unexpected error in userData function: sequence item 2: expected str instance, int found [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:557]
2024-11-04 14:53:39,094 - ERROR - Unexpected error in userData function: "Key 'title' not found. To replace missing values of 'title' with np.nan, pass in errors='ignore'" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:563]
2024-11-04 14:54:24,560 - ERROR - Unexpected error in userData function: {'id': 55791, 'title': '"Oshi no Ko" 2nd Season', 'main_picture': {'medium': 'https://cdn.myanimelist.net/images/anime/1006/143302.webp', 'large': 'https://cdn.myanimelist.net/images/anime/1006/143302l.webp'}, 'start_season': {'year': 2024, 'season': 'summer'}, 'genres': [{'id': 8, 'name': 'Drama'}, {'id': 72, 'name': 'Reincarnation'}, {'id': 42, 'name': 'Seinen'}, {'id': 75, 'name': 'Showbiz'}], 'mean': 8.64, 'rank': 78, 'rating': 'pg_13', 'studios': [{'id': 95, 'name': 'Doga Kobo'}], 'source': 'manga', 'my_list_status': {'status': 'completed', 'score': 0, 'num_episodes_watched': 13, 'is_rewatching': False, 'updated_at': '2024-10-15T05:15:36+00:00'}} has non list value 55791 for path id. Must be list or null. [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:564]
2024-11-04 14:55:34,386 - ERROR - Unexpected error in userData function: unhashable type: 'list' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:564]
2024-11-04 15:17:48,920 - ERROR - Unexpected error in userData function: "Key 'studios' not found. If specifying a record_path, all elements of data should have the path." [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:566]
2024-11-04 15:18:56,319 - ERROR - Unexpected error in userData function: "Key 'mean' not found. To replace missing values of 'mean' with np.nan, pass in errors='ignore'" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:575]
2024-11-04 15:19:10,801 - ERROR - Unexpected error in userData function: "Key 'rank' not found. To replace missing values of 'rank' with np.nan, pass in errors='ignore'" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:574]
2024-11-04 15:19:14,130 - ERROR - Unexpected error in userData function: "Key 'rank' not found. To replace missing values of 'rank' with np.nan, pass in errors='ignore'" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:574]
2024-11-04 15:26:15,154 - ERROR - Unexpected error in userData function: "Key 'mean' not found. To replace missing values of 'mean' with np.nan, pass in errors='ignore'" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:568]
2024-11-04 15:29:59,268 - ERROR - Unexpected error in userData function: name 'json_date' is not defined [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:573]
2024-11-04 15:30:10,956 - ERROR - Unexpected error in userData function: name 'json_data' is not defined [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:573]
2024-11-04 16:34:25,134 - ERROR - Unexpected error in userData function: Length mismatch: Expected axis has 13 elements, new values have 10 elements [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:554]
2024-11-04 16:35:27,543 - ERROR - Unexpected error in userData function: Length mismatch: Expected axis has 13 elements, new values have 11 elements [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:554]
2024-11-04 16:36:01,107 - ERROR - Unexpected error in userData function: Length mismatch: Expected axis has 13 elements, new values have 12 elements [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:554]
2024-11-04 16:39:16,302 - ERROR - Unexpected error in userData function: "Key 'studios' not found. If specifying a record_path, all elements of data should have the path." [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:582]
2024-11-04 16:43:05,409 - ERROR - Unexpected error in userData function: 'data' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:581]
2024-11-04 16:50:32,447 - ERROR - Unexpected error in userData function: name 'df' is not defined [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:582]
2024-11-04 16:51:51,395 - ERROR - Unexpected error in userData function: string indices must be integers, not 'str' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:587]
2024-11-04 17:06:31,495 - ERROR - Unexpected error in userData function: 'genres' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:584]
2024-11-04 17:20:53,662 - ERROR - Unexpected error in userData function: ('genres', 'rating') [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:566]
2024-11-04 17:27:49,148 - ERROR - Unexpected error in userData function: "Column(s) ['score'] do not exist" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:573]
2024-11-04 17:51:18,535 - ERROR - Unexpected error in userData function: 'score' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:589]
2024-11-04 17:54:32,770 - ERROR - Unexpected error in userData function: Length mismatch: Expected axis has 2 elements, new values have 3 elements [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:583]
2024-11-04 17:55:42,242 - ERROR - Unexpected error in userData function: "Column(s) ['genre'] do not exist" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:586]
2024-11-04 19:46:26,328 - ERROR - Unexpected error in userData function: Length mismatch: Expected axis has 4 elements, new values have 2 elements [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:574]
2024-11-04 19:48:54,016 - ERROR - Unexpected error in userData function: "['genre'] not in index" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:573]
2024-11-05 11:50:08,696 - ERROR - Unexpected error in userData function: 'ratings' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:585]
2024-11-05 11:50:24,336 - ERROR - Unexpected error in userData function: 'rating' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:585]
2024-11-05 11:59:26,272 - ERROR - Unexpected error in userData function: "['genre'] not in index" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:595]
2024-11-05 12:28:23,996 - ERROR - Unexpected error in userData function: 'average' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:609]
2024-11-05 13:46:20,464 - ERROR - Unexpected error in userData function: Length mismatch: Expected axis has 5 elements, new values have 4 elements [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:624]
2024-11-05 14:27:33,731 - ERROR - Unexpected error in userData function: 'method' object has no attribute 'index' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:626]
2024-11-05 14:28:08,801 - ERROR - Unexpected error in userData function: 'method' object has no attribute 'index' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:625]
2024-11-05 14:28:19,348 - ERROR - Unexpected error in userData function: 'method' object has no attribute 'index' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:625]
2024-11-05 14:29:17,604 - ERROR - Unexpected error in userData function: could not convert string to float: 'Doga Kobo' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:625]
2024-11-05 14:30:38,050 - ERROR - Unexpected error in userData function: Series.to_dict() got an unexpected keyword argument 'orient' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:622]
2024-11-06 14:32:09,162 - ERROR - Unexpected error in userData function: 'year' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:630]
2024-11-07 14:26:26,504 - ERROR - Unexpected error in userData function: Object of type Response is not JSON serializable [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:629]
2024-11-07 15:34:52,946 - ERROR - Error fetching weekly anime for authenticated user in userData [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:617]
2024-11-08 11:47:51,153 - ERROR - Unexpected error in userData function: name 'np' is not defined [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:629]
2024-11-08 11:49:29,767 - ERROR - Unexpected error in userData function: name 'np' is not defined [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:629]
2024-11-08 13:27:20,288 - ERROR - Unexpected error in userData function: Length mismatch: Expected axis has 12 elements, new values have 13 elements [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:629]
2024-11-11 18:48:02,618 - ERROR - Unexpected error in userData function: 'my_list_status' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:740]
2024-11-11 18:50:56,527 - ERROR - Unexpected error in userData function: "Key 'mean' not found. To replace missing values of 'mean' with np.nan, pass in errors='ignore'" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:738]
2024-11-11 18:51:32,988 - ERROR - Unexpected error in userData function: "Key 'mean' not found. To replace missing values of 'mean' with np.nan, pass in errors='ignore'" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:738]
2024-11-11 19:12:56,862 - ERROR - Unexpected error in userData function: unhashable type: 'list' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:740]
2024-11-11 19:18:37,232 - ERROR - Unexpected error in userData function: unhashable type: 'list' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:740]
2024-11-11 19:20:08,285 - ERROR - Unexpected error in userData function: unhashable type: 'list' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:740]
2024-11-11 19:20:24,251 - ERROR - Unexpected error in userData function: unhashable type: 'list' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-11 19:21:05,098 - ERROR - Unexpected error in userData function: unhashable type: 'list' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:741]
2024-11-11 19:21:17,808 - ERROR - Unexpected error in userData function: "Key 'mean' not found. To replace missing values of 'mean' with np.nan, pass in errors='ignore'" [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:741]
2024-11-12 15:57:46,817 - ERROR - Unexpected error in userData function: 'node.genres' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:710]
2024-11-12 15:59:03,626 - ERROR - Unexpected error in userData function: 'node.genres' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:710]
2024-11-12 16:03:04,224 - ERROR - Unexpected error in userData function: 'my_list_status' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:803]
2024-11-13 13:24:56,238 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
2024-11-13 13:24:56,247 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1169]
2024-11-13 13:24:56,250 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1150]
2024-11-13 13:24:56,264 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1119]
2024-11-13 13:24:56,277 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:257]
2024-11-13 13:24:56,278 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:263]
2024-11-13 13:24:56,288 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:814]
2024-11-13 13:24:56,294 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:917]
2024-11-13 13:24:56,302 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:24:56,325 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-13 13:24:56,332 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:24:56,334 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:766]
2024-11-13 13:24:56,336 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:24:56,351 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:285]
2024-11-13 13:24:56,361 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:215]
2024-11-13 13:24:56,362 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:231]
2024-11-13 13:24:56,367 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:24:56,368 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:359]
2024-11-13 13:24:56,373 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:391]
2024-11-13 13:24:56,376 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:24:56,377 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:403]
2024-11-13 13:24:56,383 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:838]
2024-11-13 13:24:56,386 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:846]
2024-11-13 13:24:56,390 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:884]
2024-11-13 13:24:56,401 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='1535363375344'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:180]
2024-11-13 13:24:56,403 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:24:56,404 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:24:56,408 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:24:56,414 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:141]
2024-11-13 13:24:56,416 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:188]
2024-11-13 13:27:13,990 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
2024-11-13 13:27:13,998 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1169]
2024-11-13 13:27:14,000 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1150]
2024-11-13 13:27:14,014 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1119]
2024-11-13 13:27:14,023 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:257]
2024-11-13 13:27:14,024 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:263]
2024-11-13 13:27:14,031 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:814]
2024-11-13 13:27:14,040 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:917]
2024-11-13 13:27:14,052 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:27:14,077 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-13 13:27:14,082 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:27:14,084 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:766]
2024-11-13 13:27:14,086 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:27:14,102 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:285]
2024-11-13 13:27:14,112 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:215]
2024-11-13 13:27:14,114 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:231]
2024-11-13 13:27:14,117 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:27:14,119 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:359]
2024-11-13 13:27:14,125 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:391]
2024-11-13 13:27:14,127 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:27:14,128 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:403]
2024-11-13 13:27:14,135 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:838]
2024-11-13 13:27:14,137 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:846]
2024-11-13 13:27:14,142 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:884]
2024-11-13 13:27:14,151 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='2634625778800'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:180]
2024-11-13 13:27:14,153 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:27:14,155 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:27:14,159 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:27:14,164 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:141]
2024-11-13 13:27:14,165 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:188]
2024-11-13 13:27:21,600 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
2024-11-13 13:27:21,611 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1169]
2024-11-13 13:27:21,613 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1150]
2024-11-13 13:27:21,626 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1119]
2024-11-13 13:27:21,633 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:257]
2024-11-13 13:27:21,634 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:263]
2024-11-13 13:27:21,641 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:814]
2024-11-13 13:27:21,649 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:917]
2024-11-13 13:27:21,656 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:27:21,681 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-13 13:27:21,685 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:27:21,686 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:766]
2024-11-13 13:27:21,689 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:27:21,705 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:285]
2024-11-13 13:27:21,715 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:215]
2024-11-13 13:27:21,716 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:231]
2024-11-13 13:27:21,720 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:27:21,722 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:359]
2024-11-13 13:27:21,729 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:391]
2024-11-13 13:27:21,731 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:27:21,733 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:403]
2024-11-13 13:27:21,739 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:838]
2024-11-13 13:27:21,742 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:846]
2024-11-13 13:27:21,746 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:884]
2024-11-13 13:27:21,754 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='2277655186512'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:180]
2024-11-13 13:27:21,756 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:27:21,758 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:27:21,762 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:27:21,768 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:141]
2024-11-13 13:27:21,769 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:188]
2024-11-13 13:27:33,307 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
2024-11-13 13:27:33,318 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1169]
2024-11-13 13:27:33,320 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1150]
2024-11-13 13:27:33,334 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1119]
2024-11-13 13:27:33,342 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:257]
2024-11-13 13:27:33,343 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:263]
2024-11-13 13:27:33,350 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:814]
2024-11-13 13:27:33,356 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:917]
2024-11-13 13:27:33,363 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:27:33,387 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-13 13:27:33,391 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:27:33,393 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:766]
2024-11-13 13:27:33,397 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:27:33,413 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:285]
2024-11-13 13:27:33,424 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:215]
2024-11-13 13:27:33,426 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:231]
2024-11-13 13:27:33,430 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:27:33,432 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:359]
2024-11-13 13:27:33,439 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:391]
2024-11-13 13:27:33,441 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:27:33,442 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:403]
2024-11-13 13:27:33,449 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:838]
2024-11-13 13:27:33,451 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:846]
2024-11-13 13:27:33,455 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:884]
2024-11-13 13:27:33,464 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='2836281952016'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:180]
2024-11-13 13:27:33,466 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:27:33,468 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:27:33,472 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:27:33,478 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:141]
2024-11-13 13:27:33,479 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:188]
2024-11-13 13:29:00,283 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
2024-11-13 13:29:00,293 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1169]
2024-11-13 13:29:00,296 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1150]
2024-11-13 13:29:00,311 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1119]
2024-11-13 13:29:00,318 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:257]
2024-11-13 13:29:00,319 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:263]
2024-11-13 13:29:00,327 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:814]
2024-11-13 13:29:00,334 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:917]
2024-11-13 13:29:00,341 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:29:00,364 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-13 13:29:00,369 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:29:00,370 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:766]
2024-11-13 13:29:00,373 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:29:00,388 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:285]
2024-11-13 13:29:00,399 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:215]
2024-11-13 13:29:00,400 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:231]
2024-11-13 13:29:00,404 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:29:00,407 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:359]
2024-11-13 13:29:00,415 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:391]
2024-11-13 13:29:00,418 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:29:00,421 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:403]
2024-11-13 13:29:00,432 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:838]
2024-11-13 13:29:00,435 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:846]
2024-11-13 13:29:00,439 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:884]
2024-11-13 13:29:00,448 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='2967412294288'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:180]
2024-11-13 13:29:00,451 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:29:00,452 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:29:00,457 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:29:00,463 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:141]
2024-11-13 13:29:00,464 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:188]
2024-11-13 13:29:09,155 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
2024-11-13 13:29:09,164 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1169]
2024-11-13 13:29:09,166 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1150]
2024-11-13 13:29:09,179 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1119]
2024-11-13 13:29:09,187 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:257]
2024-11-13 13:29:09,187 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:263]
2024-11-13 13:29:09,195 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:814]
2024-11-13 13:29:09,203 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:917]
2024-11-13 13:29:09,210 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:29:09,233 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-13 13:29:09,237 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:29:09,238 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:766]
2024-11-13 13:29:09,242 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:29:09,257 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:285]
2024-11-13 13:29:09,267 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:215]
2024-11-13 13:29:09,269 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:231]
2024-11-13 13:29:09,273 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:29:09,274 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:359]
2024-11-13 13:29:09,280 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:391]
2024-11-13 13:29:09,282 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:29:09,283 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:403]
2024-11-13 13:29:09,289 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:838]
2024-11-13 13:29:09,292 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:846]
2024-11-13 13:29:09,296 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:884]
2024-11-13 13:29:09,305 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='1508020487376'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:180]
2024-11-13 13:29:09,308 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:29:09,310 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:29:09,314 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:29:09,319 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:141]
2024-11-13 13:29:09,320 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:188]
2024-11-13 13:33:00,712 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
2024-11-13 13:33:00,721 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1169]
2024-11-13 13:33:00,724 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1150]
2024-11-13 13:33:00,737 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1119]
2024-11-13 13:33:00,746 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:257]
2024-11-13 13:33:00,747 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:263]
2024-11-13 13:33:00,756 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:814]
2024-11-13 13:33:00,762 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:917]
2024-11-13 13:33:00,769 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:33:00,793 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-13 13:33:00,797 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:33:00,798 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:766]
2024-11-13 13:33:00,803 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:33:00,817 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:285]
2024-11-13 13:33:00,826 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:215]
2024-11-13 13:33:00,828 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:231]
2024-11-13 13:33:00,832 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:33:00,833 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:359]
2024-11-13 13:33:00,840 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:391]
2024-11-13 13:33:00,843 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:33:00,843 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:403]
2024-11-13 13:33:00,851 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:838]
2024-11-13 13:33:00,854 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:846]
2024-11-13 13:33:00,857 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:884]
2024-11-13 13:33:00,871 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='2382538031520'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:180]
2024-11-13 13:33:00,874 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:33:00,878 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:33:00,881 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:33:00,889 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:141]
2024-11-13 13:33:00,890 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:188]
2024-11-13 13:33:09,711 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
2024-11-13 13:33:09,722 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1169]
2024-11-13 13:33:09,725 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1150]
2024-11-13 13:33:09,739 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1119]
2024-11-13 13:33:09,748 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:257]
2024-11-13 13:33:09,748 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:263]
2024-11-13 13:33:09,757 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:814]
2024-11-13 13:33:09,764 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:917]
2024-11-13 13:33:09,769 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:33:09,792 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-13 13:33:09,797 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:33:09,800 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:766]
2024-11-13 13:33:09,803 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:33:09,819 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:285]
2024-11-13 13:33:09,828 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:215]
2024-11-13 13:33:09,830 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:231]
2024-11-13 13:33:09,833 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:33:09,836 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:359]
2024-11-13 13:33:09,842 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:391]
2024-11-13 13:33:09,844 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:33:09,844 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:403]
2024-11-13 13:33:09,851 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:838]
2024-11-13 13:33:09,855 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:846]
2024-11-13 13:33:09,859 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:884]
2024-11-13 13:33:09,868 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='2651027574128'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:180]
2024-11-13 13:33:09,870 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:33:09,872 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:33:09,876 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:33:09,881 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:141]
2024-11-13 13:33:09,883 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:188]
2024-11-13 13:33:39,407 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
2024-11-13 13:33:39,417 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1169]
2024-11-13 13:33:39,418 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1150]
2024-11-13 13:33:39,432 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1119]
2024-11-13 13:33:39,440 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:257]
2024-11-13 13:33:39,441 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:263]
2024-11-13 13:33:39,448 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:814]
2024-11-13 13:33:39,454 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:917]
2024-11-13 13:33:39,461 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:33:39,483 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-13 13:33:39,488 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:33:39,490 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:766]
2024-11-13 13:33:39,494 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:33:39,508 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:285]
2024-11-13 13:33:39,517 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:215]
2024-11-13 13:33:39,518 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:231]
2024-11-13 13:33:39,523 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:33:39,524 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:359]
2024-11-13 13:33:39,531 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:391]
2024-11-13 13:33:39,533 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:33:39,534 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:403]
2024-11-13 13:33:39,541 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:838]
2024-11-13 13:33:39,544 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:846]
2024-11-13 13:33:39,547 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:884]
2024-11-13 13:33:39,557 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='2656398757136'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:180]
2024-11-13 13:33:39,558 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:33:39,560 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:33:39,564 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:33:39,569 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:141]
2024-11-13 13:33:39,570 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:188]
2024-11-13 13:36:28,636 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
2024-11-13 13:36:28,645 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1169]
2024-11-13 13:36:28,648 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1150]
2024-11-13 13:36:28,661 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1119]
2024-11-13 13:36:28,668 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:257]
2024-11-13 13:36:28,669 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:263]
2024-11-13 13:36:28,676 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:814]
2024-11-13 13:36:28,683 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:917]
2024-11-13 13:36:28,689 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:36:28,712 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-13 13:36:28,716 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:36:28,718 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:766]
2024-11-13 13:36:28,722 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:36:28,736 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:285]
2024-11-13 13:36:28,746 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:215]
2024-11-13 13:36:28,748 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:231]
2024-11-13 13:36:28,751 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:36:28,753 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:359]
2024-11-13 13:36:28,759 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:391]
2024-11-13 13:36:28,761 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:36:28,763 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:403]
2024-11-13 13:36:28,769 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:838]
2024-11-13 13:36:28,772 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:846]
2024-11-13 13:36:28,776 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:884]
2024-11-13 13:36:28,791 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='1735098191168'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:180]
2024-11-13 13:36:28,794 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:36:28,798 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:36:28,802 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:36:28,808 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:141]
2024-11-13 13:36:28,809 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:188]
2024-11-13 13:36:42,373 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
2024-11-13 13:36:42,383 - WARNING - State mismatch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1169]
2024-11-13 13:36:42,386 - ERROR - Error exchanging tokens in oauth [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1150]
2024-11-13 13:36:42,401 - ERROR - Error saving to db in oauth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:1119]
2024-11-13 13:36:42,410 - ERROR - Token refresh error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:257]
2024-11-13 13:36:42,411 - ERROR - Unexpected error in check_expiry:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:263]
2024-11-13 13:36:42,419 - ERROR - Unexpected error in protectedRoute:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:814]
2024-11-13 13:36:42,426 - ERROR - Unknown Error in checkSession [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:917]
2024-11-13 13:36:42,432 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:36:42,455 - ERROR - Error weekly anime as guest in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:739]
2024-11-13 13:36:42,461 - ERROR - Unexpected error in weekly_anime function: 'NoneType' object has no attribute 'decode' [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:36:42,463 - ERROR - Error fetching weekly anime for authenticated user in weekly_anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:766]
2024-11-13 13:36:42,466 - ERROR - Unexpected error in weekly_anime function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:778]
2024-11-13 13:36:42,481 - ERROR - Rate limit check error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:285]
2024-11-13 13:36:42,491 - ERROR - Database commit error:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:215]
2024-11-13 13:36:42,493 - ERROR - Unexpected error in delete_user_session:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:231]
2024-11-13 13:36:42,497 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:36:42,499 - ERROR - Error fetching plan to watch as guest [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:359]
2024-11-13 13:36:42,506 - ERROR - Error fetching plan to watch for authenticated user in plan_to_watch [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:391]
2024-11-13 13:36:42,510 - WARNING - No data found in [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:293]
2024-11-13 13:36:42,511 - ERROR - Unexpected error in plan_to_watch function:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:403]
2024-11-13 13:36:42,518 - ERROR - Error refreshing tokens for a user [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:838]
2024-11-13 13:36:42,521 - ERROR - Unexpected error in refreshUserTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:846]
2024-11-13 13:36:42,524 - ERROR - Failed to refresh tokens in refreshTokens [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:884]
2024-11-13 13:36:42,534 - ERROR - Unexpected response data in updateStatus: <MagicMock name='patch()' id='1475105016288'> [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:180]
2024-11-13 13:36:42,537 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:36:42,539 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:36:42,543 - WARNING - Invalid request data for updating anime [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:129]
2024-11-13 13:36:42,549 - ERROR - Decryption error: Decryption failed [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:141]
2024-11-13 13:36:42,551 - ERROR - Unexpected error in updateStatus:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:188]
2024-11-13 13:37:07,474 - ERROR - Database commit error in auth:  [in C:\Users\Andy\Documents\Projects\Anime Calendar\animecalendar\backend\main.py:976]
